crates/starknet-server/src/api/http/endpoints/blocks.rs

```diff
@@ -1,11 +1,39 @@
-use axum::Json;
+use axum::{Extension, Json};
 
 use crate::api::http::error::HttpApiError;
-use crate::api::http::models::{AbortedBlocks, AbortingBlocks, CreatedBlock};
-use crate::api::http::HttpApiResult;
+use crate::api::http::models::{AbortedBlocks, AbortingBlocks, BlockInfo};
+use crate::api::http::{HttpApiHandler, HttpApiResult};
 
-pub(crate) async fn create_block() -> HttpApiResult<Json<CreatedBlock>> {
-    Err(HttpApiError::GeneralError)
+pub(crate) async fn create_block(
```

Comment:
This PR apparently partially resolves this [task ](https://github.com/0xSpaceShard/starknet-devnet-rs/issues/228) ? Probably a good idea to document it as well? cc: @FabijanC 
https://0xspaceshard.github.io/starknet-devnet/docs/guide/blocks

Reply:
added in 030a4d18291edf4086acd74bc51ea88793a6bb4c